### PR TITLE
simple server name input

### PR DESCRIPTION
**Pull request by sornas**

### CODE REVIEW

File: server.sy [hunk 1]
```diff
@@ -1,10 +1,29 @@
 use game
+use text
 
 main_loop :: fn {
     n_rpc_start_server'
 
+    // Ask for server name
+    server_name := "Puck.NET"
```
Comment (FredTheDino) [inline server.sy:8]: :hurtrealbad:

File: server.sy [hunk 2]
```diff
@@ -1,10 +1,29 @@
...
+        l_update'
+        l_render'
+        text_input = l_text_input_update' server_name
+        server_name = text_input[0]
+        if text_input[1] {
+            break
+            //TODO(gu): Also handle escape here
+        }
+        //TODO(gu): draw text centered
+        text.draw_text' 0.0, 0.2, 1.0, server_name + "|", (255., 255., 255., 255.) / 255.
```
Comment (FredTheDino) [inline server.sy]: Add a flashing "|"? Can't you swap it out for a space?

File: server.sy [hunk 3]
```diff
@@ -1,10 +1,29 @@
 use game
+use text
 
 main_loop :: fn {
     n_rpc_start_server'
 
+    // Ask for server name
+    server_name := "Puck.NET"
+    l_set_text_input_enabled' true
```
Comment (FredTheDino) [inline server.sy]: We don't need to do this - do we? If we don't we could trigger escape easily

**Review by FredTheDino — APPROVED** [review]

File: server.sy (hunk 3 above)
Comment (sornas) [reply server.sy]: As discussed, otherwise the string we pass to `l_text_input_update` won't update.

**Review by EMattfolk — APPROVED** [review]

**Pull request merged**